src/components/game/PlayerArea.tsx

```diff
@@ -24,6 +27,9 @@ export const PlayerArea = ({
   isObserver = false,
   viewerIndex = 3,
 }: PlayerAreaProps) => {
+  // Get teammate reveal state
+  const isTeammateRevealed = useAppSelector(selectIsTeammateRevealed);
+
```

Comment:
Let's pass this as a prop from GameBoard. 

See how `runningSuite` is passed to PlayerArea from GameBoard